### PR TITLE
Static content collection

### PR DESCRIPTION
**Pull request by foreignfilm**

### CODE REVIEW

File: server/src/content.rs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        let path = path::to(path::ContentCategory::SuspectBackgrounds, "en");
+
+        let mut file = File::open(path)
+            .expect("suspect_backgrounds content file not found");
+
+        let mut contents = String::new();
+        file.read_to_string(&mut contents)
+            .expect("something went wrong reading the file");
+
+        return serde_json::from_str(&contents)
```
Comment (OneSadCookie) [inline server/src/content.rs]: the return here is unnecessary I think (in rust the value of any control flow is its final expression, like ruby)

File: server/src/main.rs [hunk 2]
```diff
@@ -75,7 +78,15 @@ fn main() {
                     })
             })
         });
+
+    let dump_content = warp::path("dump_content")
+        .map(|| {
+            let suspect_backgrounds = &*content::suspect_backgrounds;
```
Comment (OneSadCookie) [inline server/src/main.rs]: the `&*` should be redundant, but I'm guessing it's here to fix an error?

**Review by OneSadCookie — COMMENTED** [review]
pretty good for your first try!

File: server/src/content.rs (hunk 1 above)
Comment (foreignfilm) [reply server/src/content.rs]: ew but ok

File: server/src/main.rs (hunk 2 above)
Comment (foreignfilm) [reply server/src/main.rs]: Hmm yeah, it's my understanding that `lazy_static` creates a special type to represent the wrapped value, which then needs to be de-referenced to access the actual vector (in this case). There may be a better way to do that? This certainly doesn't work without the `&*`:

screen shot 2018-10-15 at 11 03 56 pm

File: content/en/packets/body/patient_robots.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+[
+    {
+        "vulnerability": "Internality",
+        "vulnerability_description": "You may only describe thigns that would be visible to an outside observer."
```
Comment (OneSadCookie) [inline content/en/packets/body/patient_robots.json]:
```suggestion
        "vulnerability_description": "You may only describe things that would be visible to an outside observer."
```

File: content/en/packets/moral/violent_robots.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
+[
+    {
+        "obsession": "Criminality",
+        "objective1": "Confess to a crime in response to 3 different questions.",
+        "objective2": "Accuse 2 other people of comitting crimes.."
```
Comment (OneSadCookie) [inline content/en/packets/moral/violent_robots.json]:
```suggestion
        "objective2": "Accuse 2 other people of comitting crimes."
```

File: content/en/penalties.json [hunk 5]
```diff
@@ -0,0 +1,19 @@
+[
+    {"title": "Apologize"},
+    {"title": "Swear"},
+    {"title": "Snap your fingers"},
+    {"title": "Clap audibly"},
+    {"title": "Mispronounce a word"},
+    {"title": "Hum three notes"},
```
Comment (OneSadCookie) [inline content/en/penalties.json]: I thought some of these were removed?

File: content/en/suspect_notes.json [hunk 6]
```diff
@@ -0,0 +1,28 @@
+[
+    {"title": "Disgraced archaelogist"},
+    {"title": "Maker of false animals"},
+    {"title": "Friends with the commissioner"},
+    {"title": "Renowed professor"},
+    {"title": "Reality TV contenstant"},
+    {"title": "Cult leader"},
+    {"title": "Foreign ambassador"},
+    {"title": "Imprisoned for life"},
```
Comment (OneSadCookie) [inline content/en/suspect_notes.json]: I thought some of these were removed.

File: content/en/penalties.json (hunk 5 above)
Comment (foreignfilm) [reply content/en/penalties.json]: Yeah I'm working my way through the new prompts at the moment. Will tackle these soon.

**Pull request merged**